crates/uv/src/commands/tool/uninstall.rs

```diff
@@ -67,12 +68,14 @@ pub(crate) async fn uninstall(
         }
     }
 
+    let s = if entrypoints.len() == 1 { "" } else { "s" };
     writeln!(
         printer.stderr(),
-        "Uninstalled: {}",
+        "Uninstalled {} executable{s}: {}",
```

Comment:
Are you opposed to using "executable" for user-facing copy?

Reply:
It feels a little weird, I don't have any great alternatives though. "tool entry points"? eh.

Reply:
We use tool entry points everywhere else in this output, right?

Reply:
But entry point is not as general... We also support data scripts which aren't entry points. I feel like entry point is sort of an implementation detail.

Reply:
Fair. Why not just "commands" then?

Reply:
I don't have strong feelings here.

Reply:
Cargo uses `executable`, e.g.

```
   Replacing /Users/zb/.cargo/bin/cargo-nextest
    Replaced package `cargo-nextest v0.9.70` with `cargo-nextest v0.9.72` (executable `cargo-nextest`)
```

Reply:
Ok, I'm gonna run with executable. I can change other user-facing copy too for consistency.